guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,6 +12,22 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Leapp utility does not properly enable the {Project} module for {ProjectServer}.
+Run the following command to work around the issue:
++
+----
+# subscription-manager repo-override \
+--{RepoRHEL8ServerSatelliteServerProductVersion} \
+--add=module_hotfixes:1
+----
+For {SmartProxyServer}:
```

Comment:
This sentence reads incomplete. Is this a step that is supposed to be done instead of the previous step if we are upgrading smartproxy server instead of a project server?